gtk3/sample/misc/aboutdialog2.rb

```diff
@@ -12,15 +12,6 @@
   exit
 end
 
-Gtk::AboutDialog.set_email_hook {|about, link|
```

Comment:
I don't know because I have to create an instance in order to add a signal and the show method is a class method so I think that the dialog that is shown and the dialog that is connected to the signal won't be the same. Maybe I should try to use something like mydialog.class.show.